cmd/kogito/command/shared/kogitochannels_test.go

```diff
@@ -0,0 +1,19 @@
+package shared
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_IsValidChannel(t *testing.T) {
```

Comment:
add another one to test the dev-preview channel

Reply:
add new test for dev-preview channel